delorean/interface.py

```diff
@@ -35,7 +59,13 @@ def parse(s, dayfirst=True, yearfirst=True):
         # delorean object that represents the time.
         do = Delorean(datetime=dt, timezone=UTC)
     return do
-
+try:
+    parse.__doc__ = parse.__doc__.format(mode_utc=NAIVEMODE_UTC,
```

Comment:
Why not just hard code it, its not like its going to change often enough to warrant the code.